docs/pages/blog/2021-q3-update.md

```diff
@@ -111,10 +111,10 @@ The date picker sits at the border between the core component and advanced compo
 - 🎉 We released the first stable version: [v4.0.0](https://github.com/mui/mui-x/releases/tag/v4.0.0)!
   We also released the first v5-beta, to support MUI Core v5,
   and will be working almost exclusively on the v5 release line going forward.
-- 🔎 We also added a [quick-filter demo](/x/react-data-grid/filtering/#quick-filter) to illustrate how it can be implemented,
+- 🔎 We also added a [quick-filter demo](/x/react-data-grid/filtering/quick-filter/) to illustrate how it can be implemented,
```

Comment:
A monorepo update after this should fix failing build on https://github.com/mui/mui-x/pull/9074